src/Native/configure.cmake

```diff
@@ -317,6 +317,14 @@ check_symbol_exists(
     "sys/syslimits.h"
     HAVE_OPEN_MAX)
 
+if (CMAKE_SYSTEM_NAME STREQUAL Linux)
```

Comment:
@kapilash That suggests that you should just have a higher priority check for the OS X GSS framework.  Possibly behind an if OSX check (since that's platform preference, not platform degeneration).  When looking at these ifs, think "What Would Solaris Do?"  In that it's a platform we don't currently support, but could in the eventual future.  If you'd need to write "if Solaris" (on the general assumption that it'd follow the Linux approach), we've done it wrong.